cmd/snap-confine/ns-support.c

```diff
@@ -383,6 +383,7 @@ static int sc_inspect_and_maybe_discard_stale_ns(int mnt_fd,
 		// systemd. This makes us end up in a situation where the outer base
 		// snap will never match the rootfs inside the mount namespace.
 		bool should_discard =
+		    inv->
 		    is_normal_mode ? should_discard_current_ns(base_snap_dev) :
```

Comment:
Breaking up 'inv->' and 'is_normal_mode' seems like strange formatting; can't we format it so 'inv->is_normal_mode' is together?

Reply:
No, it's indent. We would have to reformat the whole file or ignore indent from now on.